### PR TITLE
Sample Updates

### PR DESCRIPTION
**Pull request by sneakzttv**
Added contract call and private key sign/send to the UnsortedSample.cs file for ECDSA later.
Removed some magic strings and changed them to input parameters in UnsortedSample.cs.
Updated tests.

### CODE REVIEW

File: Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/Unsorted/ContractSendBehaviour.cs [hunk 1]
```diff
@@ -8,6 +8,10 @@ public class ContractSendBehaviour : SampleBehaviour
         public string method = "addTotal";
         public string abi = "[ { \"inputs\": [ { \"internalType\": \"uint8\", \"name\": \"_myArg\", \"type\": \"uint8\" } ], \"name\": \"addTotal\", \"outputs\": [], \"stateMutability\": \"nonpayable\", \"type\": \"function\" }, { \"inputs\": [], \"name\": \"myTotal\", \"outputs\": [ { \"internalType\": \"uint256\", \"name\": \"\", \"type\": \"uint256\" } ], \"stateMutability\": \"view\", \"type\": \"function\" } ]";
         public string contractAddress = "0x7286Cf0F6E80014ea75Dbc25F545A3be90F4904F";
+        public object[] args =
+        {
+            1
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/Unsorted/ContractSendBehaviour.cs:13]: What does that number 1 actually represent?

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/Unsorted/ContractSendBehaviour.cs:13]: The amount to increase the mapping by like so:

`// SPDX-License-Identifier: MIT
pragma solidity ^0.8.19;

contract AddTotal {
    uint256 public myTotal = 0;

    function addTotal(uint8 _myArg) public {
        myTotal = myTotal + _myArg;
    }
}`

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/Unsorted/ContractSendBehaviour.cs:13]: I should probably refactor the object so it initialized from a public integer so it's not sitting there as a magic number too. BRB.

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/Unsorted/ContractSendBehaviour.cs:13]: done

File: Packages/io.chainsafe.web3-unity/Tests/Runtime/MiscTests.cs [hunk 2]
```diff
@@ -64,8 +64,13 @@ public override IEnumerator Setup()
     public IEnumerator TestContractSend()
     {
         config.TestResponse = "0x9de3bb69db4bd93babef923f5da1f53cdb287d9ebab9b4177ba2fb25e6a09225";
-
-        var sendContract = _sample.ContractSend(ContractSendMethodName, Abi, ContractAddress);
+        
+        object[] args =
+        {
+            1
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity/Tests/Runtime/MiscTests.cs:70]: Is this also the increase amount?

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Tests/Runtime/MiscTests.cs:70]: ah test cases, yep, updating now.

**Review by kantagara — APPROVED** [review]
LGTM!

**Review by juans-chainsafe — CHANGES_REQUESTED** [review]
I get the following build error:

`/Users/juanmanuelspoleti/Desktop/workspace/web3.unity/Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthWalletExtensions.cs(50,100): error CS1026: ) expected`

**Comment by RyRy79261** [comment]
@juans-chainsafe Theres a fix in @sneakzttv latest goerli PR that is approved that should resolve that issue

**Comment by sneakzttv** [comment]
should work now @juans-chainsafe

**Review by juans-chainsafe — APPROVED** [review]
Samples working correctly!

**Pull request merged**